RecoHI/HiJetAlgos/plugins/HiPFCandCleaner.h

```diff
@@ -0,0 +1,22 @@
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+
+class HiPFCandCleaner : public edm::EDProducer {
```

Comment:
this should be changed to a thread-aware producer
https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes
It looks like this can be edm::global